democracy_club/apps/hermes/migrations/0010_add_past_post_authors.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+from django.db import migrations, models
+import csv
+
+# Get the blog object by slug (if it exists, again, because local)
+def get_post_by_slug(slug):
+    try:
+        models.Post.objects.get(slug=slug)
+    except models.Post.DoesNotExist:
+        return None
+
+
+# Get the user object by username
+def get_user_by_username(username):
+    try:
+        models.User.objects.get(username=username)
+    except models.User.DoesNotExist:
+        return None
+
+
+# Post.authors.add(user)
+def get_authors_for_past_posts(apps, schema_editor):
+    with open("/blog-authors-2022-06-28.csv") as csvfile:
+        Post = apps.get_model("hermes", "Post").objects.all()
```

Comment:
This is a way of getting the _model_ not the queryset. So you want `Post = apps.get_model("hermes", "Post")` and then pass that object in to the `get_post_by_slug` function.